data/json/itemgroups/SUS/office.json

```
@@ -0,0 +1,62 @@
+[
+  {
+    "id": "SUS_office_desk",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for an office desk containing documents and electronics for work and some desk clutter.",
+    "subtype": "collection",
+    "entries": [
+      { "item": "eink_tablet_pc", "prob": 20 },
+      { "item": "laptop", "prob": 60 },
+      { "item": "usb_drive", "prob": 20 },
+      { "item": "mobile_memory_card_used", "count": [ 1, 4 ], "prob": 15 },
+      { "item": "mobile_memory_card_encrypted", "count": [ 1, 2 ], "prob": 15 },
+      { "item": "file", "count": [ 1, 8 ], "prob": 50 },
+      { "item": "paper", "count": [ 2, 9 ], "prob": 60 },
+      { "item": "pen", "prob": 95 },
+      { "item": "stapler", "prob": 60 },
+      { "item": "scissors", "prob": 35 },
+      { "item": "phonebook", "prob": 10 },
+      { "item": "family_photo", "prob": 10 },
+      { "item": "coin_quarter", "prob": 10 },
+      { "group": "oa_discarded_news", "count": [ 2, 12 ], "prob": 20 }
+    ]
+  },
+  {
+    "id": "SUS_office_filing_cabinet",
+    "type": "item_group",
+    "//": "SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given storage furniture.",
+    "//2": "This group is for an office filing cabinet containing folders and documents.",
+    "subtype": "collection",
+    "entries": [
+      { "item": "file", "count": [ 1, 15 ], "prob": 50 },
+      { "item": "paper", "count": [ 20, 300 ], "prob": 60 },
```

```suggestion
      { "item": "paper", "count": [ 2, 30 ], "prob": 60 },
```
Each count of paper is 150 sheets, 300*150 for 45 000 sheets is probably a bit much?